2020-January.html

```diff
@@ -0,0 +1,247 @@
+                        "alt": "Hosted by Castor"
+                    },
+                    "hoster-next": {
+                        "src": "img/hoster-logos/castor.svg",
+                        "alt": "Castor"
+                    },
+                    "speaker-face": {
+                        "src": "img/speakers/thijs_de_maa.jpeg",
+                        "alt": "Speaker: Thijs de Maa"
+                    }
+                }
+            };
+        </script>
+    </head>
+
+    <body>
+
+        <div class="reveal">
+
+            <div class="slides">
+                <section>
+                    <img src="img/logo-transparent-background.png" alt="AmsterdamPHP">
+                    <h4 id="date"></h4>
+
+                    <div class="two-columns" style="margin-top: 4em;">
+                        <div>
+                            <p>Hosted by</p>
+                            <img id="hoster-small" class="hoster" src="" alt="">
+                        </div>
+
+                        <div>
+                            <ul class="pointless">
+                                <li>19h00 - 19h30: Welcome &amp; Drinks</li>
+                                <li>19h30 - 19h35: Opening</li>
+                                <li>19h35 - 20h30: Talk</li>
+                                <li>20h30 - 22h00: Raffle &amp; Social</li>
+                            </ul>
+                        </div>
+                    </div>
+                </section>
+
+                <section>
+                    <h1>Tonight's Sponsors</h1>
+                    <img id="hoster-large" class="hoster big" src="" alt="">
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">Tonight's Prizes</h1>
+                </section>
+
+                <section>
+                    <h1>Books!</h1>
+
+                    <img class="hoster big" src="img/hoster-logos/messagebird-full.svg" alt="Sponsor: MessageBird">
+                    <div class="fragment grow hoster-hiring">
+                        We're hiring!
+                    </div>
+                </section>
+
+                <!-- <section data-background="img/pagination_promo.jpg">
+                </section> -->
+
+                <section>
+                    <h1>Tonight's Speaker</h1>
+
+                    <div class="two-columns" style="margin-top: 4em;">
+                        <div>
+                            <img id="speaker-face" class="speaker" src="" alt="">
+                        </div>
+
+                        <div>
+                            <h3 id="speaker-name" class="fancy"></h3>
+                        </div>
+                    </div>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">Conferences and Events</h1>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">PHP Benelux</h1>
```

Comment:
Wasn't this conf sold out? If so, please remove it.

Reply:
On my todo list to check

Reply:
They stopped selling tickets yesterday actually, so I've removed it